definition/ILivechatDepartment.ts

```diff
@@ -12,6 +12,7 @@ export interface ILivechatDepartment {
 	numAgents: number;
 	_updatedAt?: Date;
 	businessHourId?: string;
+	fallbackForwardDepartment: string;
```

Comment:
Isn't this an optional setting? Atleast that's what it looks on the Department Edit screen